flows/engine/config.go

```diff
@@ -1,60 +1,38 @@
 package engine
 
 import (
-	"encoding/json"
-
 	"github.com/nyaruka/goflow/flows"
 )
 
 // the configuration options for the flow engine
 type config struct {
+	maxFieldLength          int
+	maxResultLength         int
```

Comment:
Is it useful for these ever being different? Results and contact values seem so closely tied that this seems potentially confusing to have as an option.

Reply:
Hmm maybe we should have a very generic `maxLength` value and apply it all over the place? Am thinking about other fields like `contact.name`, `result.category` and a nefarious user who wants to stuff huge data blobs somewhere.

Reply:
Hrmm.. ya, how does this relate to say webhook results? That's part of the `extra` for that value right so wouldn't apply there?

It just seems like when these are different behavior can get really weird for users who love doing bizarre things like passing values through contact fields etc..

Reply:
Yeah for webhooks we read up to 10000 bytes and store that as extra on the result.

I could be persuaded that we get rid of `EngineConfig` completely at this point. Made a lot more sense having it when we had a stand alone flow server. We put `maxLength` (?) on `Environment` and we apply to contact.name, contact.field.value, result.value, result.category etc.

Reply:
I could see `disableWebhooks` still being a thing on the engine config, at the very least for testing but ya, that'd be my preference.

Reply:
Maybe just `maxValueLength` (contact values and result values?)

Reply:
If `disableWebhooks` and is the only thing left on `EngineConfig` then I think that can just become a static var in the module that uses it (we're not going to be using it) - and then we can stop passing `EngineConfig` instances around everywhere.

Reply:
WFM